styles/MainInfo.module.css

```diff
@@ -1,29 +1,39 @@
+.contenedorCity {
+    @apply flex justify-between;
+}
+
+.contenedorCountry {
```

Comment:
Why mix spanish with english?

Why not only use `cityContainer` and `countryContainer`

Reply:
use English